src/reflect/scala/reflect/io/AbstractFile.scala

```diff
@@ -52,7 +52,7 @@ object AbstractFile {
    */
   def getURL(url: URL): AbstractFile =
     if (url.getProtocol == "file") {
-      val f = new java.io.File(url.getPath)
+      val f = new java.io.File(url.toURI())
```

Comment:
I'd also looked with suspicion at this line when trying to figure out what was happening, but couldn't come up with an input on windows.